telegram-bot/src/main/java/com/exadel/telegrambot/bot/utils/Security.java

```diff
@@ -0,0 +1,5 @@
+package com.exadel.telegrambot.bot.utils;
+
+public interface Security {
```

Comment:
the same

Reply:
in this class I can write here telegram token only and manager or admins tokens